ci/Dockerfile

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 
-ARG CUDA_VERSION=11.8.0
+ARG CUDA_VERSION=12.9.1
```

Comment:
Should we test with lowest cuda 12 version supported by rapids?  That would be 12.0 .   Not yet seeing 12.9 [officially supported](https://docs.rapids.ai/install/#selector) but I guess 25.08 will have that.